arangod/RocksDBEngine/RocksDBCollection.cpp

```diff
@@ -680,6 +685,8 @@ Result RocksDBCollection::truncate(transaction::Methods& trx, OperationOptions&
     _meta.adjustNumberDocuments(seq, /*revision*/ newRevisionId(),
                                 -static_cast<int64_t>(numDocs));
 
+    _statistics._numWrite.count(numDocs);
```

Comment:
Here we don't actually write documents, but we place a DeleteRange, which is not the same as placing x remove operations.
In addition if someone wants to calculcate the write operations total time divided by the total number of write operations, then the results would be much skewed by us counting the number of writes here but no timings.
I suggest we don't increase `_numWrite` here at all.

Reply:
the idea was to record deletions as writes. in a sense comparable to read and write operations on a posix file system, where deletions, truncations, modifications and creations are all counted as write.